src/common/POLFields/DetailsFields/FieldAcquisitionMethod.js

```diff
@@ -3,18 +3,13 @@ import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 
 import {
-  ACQUISITION_METHOD,
-  FieldSelectFinal,
+  FieldSelectionFinal,
 } from '@folio/stripes-acq-components';
 
-const ACQUISITION_METHOD_OPTIONS = Object.keys(ACQUISITION_METHOD).map((key) => ({
-  labelId: `ui-orders.acquisition_method.${key}`,
-  value: ACQUISITION_METHOD[key],
-}));
-
-const FieldAcquisitionMethod = ({ disabled, required }) => (
-  <FieldSelectFinal
-    dataOptions={ACQUISITION_METHOD_OPTIONS}
+const FieldAcquisitionMethod = ({ acquisitionMethods, disabled, required }) => (
```

Comment:
seems acquisitionMethods could be fetched inside `FieldAcquisitionMethod` component using react-query and avoid props drilling. what do you think?

Reply:
Looks like yes, add `useAcqMethods` hook